OpenRA.Game/Support/ExceptionHandler.cs

```diff
@@ -38,7 +38,7 @@ public static void HandleFatalError(Exception ex)
 			if (Game.OrderManager != null && Game.OrderManager.World != null && Game.OrderManager.World.Map != null)
 			{
 				var map = Game.OrderManager.World.Map;
-				Log.Write("exception", $"on map {map.Uid} ({map.Title} by {map.Author}).");
+				Log.Write("exception", $"on map {((Map)map).Uid} ({((Map)map).Title} by {((Map)map).Author}).");
```

Comment:
also feels like a missing map interface i.e. IMapInfo { Uid, Tititle, Author }, IMap.GetMapInfo() or GetMapAttributes().

optionally return them in a dictionary as key value pairs.

Reply:
The purpose of interfaces is to either A) Provide alternative implementations, or B) Provide the option to not implement the interface at all and not use the subsequent functionality. Since all maps have Titles and Authors, and all maps will implement a Title and Author in the same way, I don't see the value in adding an interface here. Perhaps in a later PR if there is a case for it, but I don't think this is critical to the current refactoring plan.